snapcraft/tests/test_yaml.py

```diff
@@ -346,6 +347,107 @@ def test_config_expands_filesets(self, mock_loadPlugin):
         })
 
 
+class TestYamlEnvironment(tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        dirs.setup_dirs()
+
+        tempdirObj = tempfile.TemporaryDirectory()
+        self.addCleanup(tempdirObj.cleanup)
+        os.chdir(tempdirObj.name)
```

Comment:
Ah good find! I was copying what was already there-- I've fixed the rest of the tests as well.